outport/notifier/httpClientWrapper.go

```diff
@@ -13,6 +13,11 @@ const (
 	minRequestTimeoutSec = 1
 	contentTypeKey       = "Content-Type"
 	contentTypeValue     = "application/json"
+
+	payloadVersionKey = "version"
+	// set http header version to 1; notifier connector will have to handle
+	// previous payload type if no version header is set
+	payloadVersionValue = "1"
```

Comment:
i've hardcoded it here since http integration will be removed anyway in the next release, so there is no need to adapt  the `EventNotifierConnector` config section for this